### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by nazardudnyk**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+            class="nav__link
+            is-active">Apple</a>
```
Comment (VitaliyBodnar) [inline src/index.html:20]: code is not aligned, check other places 
```suggestion
            <a 
              href="#apple"
              class="nav__link
              is-active"
            >Apple</a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 10 50px;
+  justify-content: space-between;
+  flex-shrink: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
```
Comment (VitaliyBodnar) [inline src/style.css:17]: normal is a default value for the font-style, so it's redundant

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
- missed demo link
Screenshot 2023-09-21 at 19 32 50

**Review by lerastarynets — CHANGES_REQUESTED** [review]
well done in fixing comments, but you still haven't deployed the project and test report, pls fix)

**Comment by Nazar-Dudnyk** [comment]
> well done in fixing comments, but you still haven't deployed the project and test report, pls fix)

file:///Users/nazardudnyk/projects/layout_moyo-header/backstop_data/html_report/index.html

http://192.168.0.102:5500/src/index.html

**Pull request closed without merging**